lms/static/sass/shared/_footer.scss

```diff
@@ -190,6 +190,10 @@
       }
     }
   }
+
+  @media print {
```

Comment:
@jolyonb I would echo the suggested change to only hide a part of the footer instead of the whole thing. By scoping your rule changes you can make the edx.org footer printable.

.footer-about --> add print media query  that makes it span 12 columns instead of 6, and .footer-nav plus .footer-follow would get the display none extend. You could also then go further down and hide the longer about us text section, plus the links to Terms of Service, Privacy Policy, and Accessibility Policy links for the print view as well. 

It is a few more rules but you shouldn't need to edit the template.